test/vector-tile.test.js

```diff
@@ -3095,7 +3095,7 @@ describe('mapnik.VectorTile ', function() {
                     "id":1,
                     "type":3,
                     "geometry":[
-                        [[2784,1776],[1976,1776],[1976,992],[2784,992],[2784,1776]]
+                        [[1976,1776],[1976,992],[2784,992],[2784,1776],[1976,1776]]
```

Comment:
Not a reversed ring direction, just a shift in the starting position of the rings.